plugin.go

```diff
@@ -48,6 +54,8 @@ type Plugin struct {
 	forbiddenExtensions map[string]struct{}
 	// opentelemetry
 	prop propagation.TextMapPropagator
+
+	gzip bool
```

Comment:
option without explanation comment. Also I think this is a bad name for the option, because what a `gzip=true` should do in the `gzip` middleware?